basic/token2.s

```diff
@@ -187,6 +214,22 @@ brktxt	.byt $d
 err30	.byt "BREAK",0,$a0 ;shifted space
 err31	.byt "NOT GRAPHICS MOD", 'E'+$80
 errngm	=31
+err32	.byt "INVALID CHANNE", 'L'+$80
```

Comment:
Yeah, my only feedback is that if we're tight on bytes we could replace "INVALID" with "BAD"

Reply:
Removed additional words altogether ;)

Reply:
Yes! Removing the superfluous adjectives altogether! Save the bytes! <3